pkg/ctl/create/cluster.go

```diff
@@ -135,11 +140,6 @@ func doCreateCluster(cmd *cmdutils.Cmd, ngFilter *filter.NodeGroupFilter, params
 		}()
 	}
 
-	ctl, err := cmd.NewCtl()
```

Comment:
The above logger initialisation breaks now that newCtl happens before this. NewCtl contains a `once` for the logger. I'm working on the fix.

Reply:
Fixed.